game_patch/hud/multi_scoreboard.cpp

```diff
@@ -53,14 +53,26 @@ int draw_scoreboard_header(int x, int y, int w, rf::NetGameType game_type, bool
 
     // Draw Game Type name
     if (!dry_run) {
-        const char* game_type_name;
-        if (game_type == rf::NG_TYPE_DM)
-            game_type_name = rf::strings::deathmatch;
-        else if (game_type == rf::NG_TYPE_CTF)
-            game_type_name = rf::strings::capture_the_flag;
-        else
-            game_type_name = rf::strings::team_deathmatch;
-        rf::gr::string_aligned(rf::gr::ALIGN_CENTER, x_center, cur_y, game_type_name);
+        const auto get_game_type_name = [] (const rf::NetGameType game_type) {
```

Comment:
please make this a standalone function, there's no reason to make it a lambda

Reply:
Why bother? It is just to make up for the fact that `if` is not an expression.

Reply:
Shorter functions are more readable and this fragment is very easy to split out